packages/core/src/internal/interfaces/index.ts

```diff
@@ -17,6 +17,8 @@ export type StatusTypes = 'neutral' | 'info' | 'success' | 'warning' | 'danger';
 // MISC
 export type AriaLivePolitenessSettings = 'off' | 'polite' | 'assertive';
 
+export type AriaBooleanAttributeValues = 'true' | 'false';
```

Comment:
Why are we making a distinction here with the string boolean values for aria?

Reply:
If you use a boolean type for the property, it will conditionally add the attribute if the value is true (much like `disabled`, `checked`, etc).

But for `aria-` attributes we need an explicit string value of `true` or `false` assigned to the attribute (e.g. `aria-selected='false'`).

Are there other non `aria-` attributes that work the same way? Happy to name the type differently if so.

Reply:
No changes needed here. I just wanted to understand the why.